addons/rating/models/mail_message.py

```diff
@@ -25,3 +25,12 @@ def _search_rating_value(self, operator, operand):
             ('message_id', '!=', False)
         ])
         return [('id', 'in', ratings.mapped('message_id').ids)]
+
+    def message_format(self, format_reply=True):
+        message_values = super().message_format(format_reply)
+        ratings = self.env['rating.rating'].sudo().search([('message_id', 'in', self.ids), ('consumed', '=', True)])
```

Comment:
Actually there is no need to search for rating on most models .. only those inheriting from the rating mixin should do that. Otherwise you add queries where it is not necessary.